pkg/scalers/azure_servicebus_scaler.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	servicebus "github.com/Azure/azure-service-bus-go"
 
-	"github.com/Azure/azure-amqp-common-go/v2/auth"
+	"github.com/Azure/azure-amqp-common-go/v3/auth"
```

Comment:
curious about the update. Is it intentional?

Reply:
I tried to update as much dependencies to the latest versions as possible. This was brought by updating some of the Azure deps. I was forced to update this particular import due to changes in the api. We can try to revert this, if you think that it is a problem.